examples/lfric/eg17/full_example_netcdf/README.md

```diff
@@ -1,13 +1,8 @@
 # Standalone LFRic Example with NetCDF
 
-This directory contains a stand-alone LFRic code example.
-It is just a proof-of-concept to evaluate the feasibility and
-required effort for a stand-alone LFRic example. This
-examples adds support for reading in the global mesh from a
-NetCDF file.
-
 This example contains the minimum infrastructure required to create
-a field, and call a simple kernel on this field. The following 
+a field, and call a simple kernel on this field. The grid is read
+from the NetCDF file ``mesh_BiP128x16-400x400.nc``. The following 
```

Comment:
``mesh_BiP128x16-400x100.nc``, as you rightly pointed out 😄

Reply:
Renamed.